src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -8482,12 +8498,6 @@ private bool IsPossibleLocalDeclarationStatement(bool isGlobalScriptLevel)
             return IsPossibleFirstTypedIdentifierInLocalDeclarationStatement(isGlobalScriptLevel);
         }
 
-        private bool IsPossibleScopedKeyword(bool isFunctionPointerParameter)
-        {
-            using var _ = this.GetDisposableResetPoint(resetOnDispose: true);
```

Comment:
for those keeping count, this made for a triple nested reset point.  We now just have one reset point.